nexus/users/models.py

```diff
@@ -4,6 +4,8 @@
 from django.db import models
 from django.utils import timezone
 from django.utils.translation import ugettext_lazy as _
+from extended_choices import Choices
```

Comment:
Why do we need `extended_choices`?

Reply:
So that in the code, we can easily refer to these choices by name which make it's more readable.

Reply:
Okay. then we should be using these throughout the application for all the other APIs we have created.